qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -33,11 +33,13 @@ monitor_url() {
   done
 }
 
+# Create pre-upgrade user
 # push app in subshell to avoid changing directory
 (
   cd ci/sample-apps/go-env
   cf api --skip-ssl-validation "https://api.${DOMAIN}"
   cf login -u admin -p changeme -o system
+  cf create-user pre-upgrade-user pre-upgrade-user
```

Comment:
Move this and other lines unrelated to pushing the app out of the subshell for clarity, since we're using that logged-in state elsewhere.